circuits/credentialAtomicQuerySigOnChainSmt.circom

```diff
@@ -0,0 +1,14 @@
+pragma circom 2.0.0;
+
+include "lib/query/credentialAtomicQuerySigOnChainSmt.circom";
+
+component main{public [userStateInOnChainSmtRoot,
+                        userNullifier,
```

Comment:
userNullifier  should be output, and should not be present as input

Reply:
I'll put it to outputs.